assets/src/apps/authoring/Accordion/Accordion.tsx

```diff
@@ -1,37 +1,52 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable react/jsx-key */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable react/prop-types */
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 const Accordion: React.FC<any> = (props) => {
   const tabs: [] = props.tabsData.tabs;
+  const clickHandler = (tabid: any) => {
+    if (expandedCards.includes(tabid))
+      setExpandedCards(expandedCards.filter((i: any) => i !== tabid));
+    else setExpandedCards([...expandedCards, tabid]);
+  };
+  const [expandedCards, setExpandedCards] = useState<any>([]);
+  useEffect(() => {
+    const tabsId = tabs.map((tab: any) => tab.id);
+    setExpandedCards(tabsId);
+  }, []);
   return (
-    <div className="accordion">
+    <div className='accordion'>
       {tabs.map((tab: any) => (
-        <div className="card">
```

Comment:
react will complain in the console if you don't specify a key on the first element of a `.map()`.  refactor to: 
```
{tabs.map((tab: any, index: number) => ( 
   <div key={index} className="card">
   ...
```
and check that you've got that for all `.map()`s ...